UPGRADE.rst

```diff
@@ -7,12 +7,37 @@ what broke and how to safely migrate to newer versions.
 From 0.15 to 0.16
 -----------------
 
+- ``tchannel.TChannel.register`` no longer mimicks ``tchannel.tornado.TChannel.register``,
+  instead it exposes the new server API like so:
+
+  Before:
+
+  .. code:: python
+
+      from tchannel import TChannel
+
+      @tchannel.register('endpoint', 'json')
+      def endpoint(request, response, proxy):
+          response.write({'resp': 'body'})
+
+
+  After:
+
+  .. code:: python
+
+      from tchannel import TChannel, Response
+
+      @tchannel.json.register
+      def endpoint(request):
```

Comment:
It could almost go out as a patch since it will fix tracing for this new api (this PR breaks it)